docs/source/user_guide/pipeline-components.md

```diff
@@ -192,6 +192,24 @@ Note: You must specify all property values, not only the ones that you want to m
 
 Refer to section [Configuration properties](#configuration-properties) for parameter descriptions.
 
+#### Exporting component catalogs
+
+To export component catalogs:
+
+```bash
+elyra-metadata export component-catalogs \
+	--schema-name="url-catalog" \
```

Comment:
```suggestion
	--schema_name="url-catalog" \
```

Reply:
While completely correct (there are different schema names defined in this namespace), we should keep it simple and not mention the `--schema_name` option in the example or text below. Given that we are positioning export as the means to create a backup, I don't believe it's likely that most users would want to export, for example, only URL catalog entries. The built-in command help calls out that it is possible to filter by schema name.

Reply:
Removed all mentions of `--schema-name` parameter from the documentation as discussed via chat.